src/App.js

```diff
@@ -1,23 +1,19 @@
-import logo from './logo.svg';
-import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import Layout from './components/Layout';
+import Rockets from './pages/Rockets';
+import Missions from './pages/Missions';
+import Profile from './pages/Profile';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Routes>
+        <Route path="/" element={<Layout />}>
+          <Route index element={<Rockets />} />
+          <Route path="/Missions" element={<Missions />} />
+          <Route path="/profile" element={<Profile />} />
+        </Route>
```

Comment:
- [ ] Kindly adjust your layout in such a way that your header and the rest of the body content should align properly on the left and right (both) hand sides. Right now, your body content is taking more width and going outside the alignment of the `header` component. You should fix the style because it is the part of requirement to follow the `wireframe` design and also it will help you to become a detail-oriented developer.

### Design Comparison
<table>
<tr>
<th>Microverse Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-10-07 12-17-14](https://github.com/AbuTalha3/react_group_project/assets/66476329/3d37d9e8-4b6d-4804-ac8d-6fba270e5412)

</td>
<td>

![Screenshot from 2023-10-07 12-16-50](https://github.com/AbuTalha3/react_group_project/assets/66476329/7fe004a1-b2ec-4244-9e01-495316bc58d5)

</td>
</tr>
</table>